cpp/src/io/parquet/experimental/hybrid_scan_helpers.hpp

```diff
@@ -50,53 +50,6 @@ struct metadata : private metadata_base {
 };
 
 class aggregate_reader_metadata : public aggregate_reader_metadata_base {
- private:
-  /**
-   * @brief Materializes column chunk dictionary pages into `cuco::static_set`s
-   *
-   * @param dictionary_page_data Dictionary page data device buffers for each input row group
-   * @param input_row_group_indices Lists of input row groups, one per source
-   * @param total_row_groups Total number of row groups in `input_row_group_indices`
-   * @param output_dtypes Datatypes of output columns
-   * @param dictionary_col_schemas schema indices of dictionary columns only
-   * @param stream CUDA stream used for device memory operations and kernel launches
-   *
-   * @return A flattened list of `cuco::static_set_ref` device buffers for each filter column
-   * across row groups
-   */
-  [[nodiscard]] std::vector<rmm::device_buffer> materialize_dictionaries(
```

Comment:
Removed these two function declarations for now since their signature will be changed in an upcoming PR. The same PR will add their impl as well so it would be appropriate to have both changes altogether there.